src/main/java/eu/webeid/security/validator/AuthTokenValidatorBuilder.java

```diff
@@ -127,6 +127,38 @@ public AuthTokenValidatorBuilder withOcspRequestTimeout(Duration ocspRequestTime
         return this;
     }
 
+    /**
+     * Sets the allowed time skew in milliseconds for OCSP response's thisUpdate and nextUpdate times.
+     * This value is used to tolerate slight discrepancies between the system clock and the OCSP responder's clock.
```

Comment:
... or special update times. Explain why the default is 15 minutes - Finnish OCSP was using CRLs that were updated every 15 minutes.